scripts/build.go

```diff
@@ -0,0 +1,61 @@
+// +build none
+
+package main
+
+import (
+	"fmt"
+	"log"
```

Comment:
Any reason why using vanilla golang `log` instead of `logrus` in the main?

Reply:
this is not the main, its the build script. If we try to use logrus, nothing will be printed.